islandora_citations.routing.yml

```diff
@@ -40,11 +40,15 @@ entity.islandora_citations.add_from_file:
     _permission: 'administer islandora_citations'
   options:
     _admin_route: TRUE
-    
+
 islandora_citations.tab.node:
   path: '/admin/structure/types/manage/{node_type}/citations-map-configuration'
   defaults:
     _title: 'Citations Map Configuration'
     _controller: 'islandora_citations.controller:provideArguments'
   requirements:
     _permission: 'administer islandora_citations'
+  options:
+    parameters:
+      node_type:
+        type: entity:node_type
```

Comment:
I _think_ this could go away? As in, Drupal's default parameter conversion should account for this, with the type hinting now on the controller method?... I think?

Reply:
```suggestion
```
Might as well remove it then; I checked in my local env and removing these lines didn't appear to cause any issues; the mapping config at this route renders fine with or without these lines.